processor/routingprocessor/logs_test.go

```diff
@@ -273,6 +273,65 @@ func TestLogs_AreCorrectlySplitPerResourceAttributeRouting(t *testing.T) {
 	)
 }
 
+func TestLogsAreCorrectlySplitPerResourceAttributeWithTQL(t *testing.T) {
+	defaultExp := &mockLogsExporter{}
+	lExp := &mockLogsExporter{}
+
+	host := &mockHost{
+		Host: componenttest.NewNopHost(),
+		GetExportersFunc: func() map[config.DataType]map[config.ComponentID]component.Exporter {
+			return map[config.DataType]map[config.ComponentID]component.Exporter{
+				config.LogsDataType: {
+					config.NewComponentID("otlp"):              defaultExp,
+					config.NewComponentIDWithName("otlp", "2"): lExp,
+				},
+			}
+		},
+	}
+
+	exp := newLogProcessor(zap.NewNop(), &Config{
+		DefaultExporters: []string{"otlp"},
+		Table: []RoutingTableItem{
```

Comment:
This test could be expanded to have two TQLs, with datapoints matching both expressions.